libs/core/src/lib/shellbar/user-actions-menu/directives/user-actions-menu-item.directive.ts

```diff
@@ -0,0 +1,91 @@
+import {
+    Directive,
+    EventEmitter,
+    HostListener,
+    Input,
+    OnDestroy,
+    Optional,
+    Output
+} from '@angular/core';
+import { finalize } from 'rxjs/operators';
+
+import { DialogContentType, DialogRef, DialogService } from '../../../dialog/public_api';
```

Comment:
can we import this directly,[ instead of public_api](https://github.com/SAP/fundamental-ngx/pull/4502#pullrequestreview-578270438)?

Reply:
@kavya-b Why is it bad to import from public_api?

Reply:
there was an issue sometime back where in ng10, using public_api for imports was causing some errors while running `ng xi18n`. Here is the PR where that issue was fixed: [link](https://github.com/SAP/fundamental-ngx/pull/4889)